src/main/java/nl/novi/techiteasy/exceptions/ExceptionController.java

```diff
@@ -15,8 +15,8 @@ public ResponseEntity<Object> handleRecordNotFoundException(RecordNotFoundExcept
     }
 
     @ExceptionHandler(Exception.class)
-    public ResponseEntity<Object> handleGeneralException() {
-        String message = "Unexpected error";
+    public ResponseEntity<Object> handleGeneralException(Exception ex) {
+        String message = "Unexpected error " + ex.getMessage();
```

Comment:
Dit is al wel een goede toevoeging, maar je merkte vandaag in de huiswerkles dat het tijdens development niet altijd handig is om deze Exception.class af te vangen, omdat je dan niet duidelijk te zien krijgt in de console welke exception er opgegooid wordt en wat het probleem is.